src/open_inwoner/accounts/models.py

```diff
@@ -23,7 +22,11 @@
 
 from .choices import ContactTypeChoices, LoginTypeChoices, StatusChoices, TypeChoices
 from .managers import ActionQueryset, DigidManager, UserManager, eHerkenningManager
-from .query import InviteQuerySet, MessageQuerySet, UserQuerySet
+from .query import InviteQuerySet, MessageQuerySet
+
+
+def generate_uuid_photo_name(instance, filename):
+    return "photo_{uuid}/{filename}".format(uuid=uuid4(), filename=filename)
```

Comment:
Let's use a sub-directory for the profile pictures, and I think you should only use the extension from the filename (because the name if the file could be personal data). 

So `profile/ea78fd4c-6708-4248-a6e3-91ff9e907a20.jpg`